### ISSUE TITLE
[Graphics] Segmentation fault

### ISSUE THREAD

**Issue by Peiprjs** [arf20/arftracksat#2]
```┌─[✗]─[peipr@P315-PT]─[~/arftracksat/src/build]
└──╼ $make && sudo make install && arftracksat 
[100%] Built target arftracksat
[100%] Built target arftracksat
Install the project...
-- Install configuration: ""
-- Up-to-date: /usr/local/bin/arftracksat
-- Installing: /usr/local/etc/arftracksat/config.json
arftracksat by arf20
Get:0 http://celestrak.com/NORAD/elements/weather.txt
Get:1 http://celestrak.com/NORAD/elements/noaa.txt
Satellites loaded
Setup done, entering loop...
Loading GeoJSON map...
Map loaded [5127 points]
Segmentation fault
```
Esto con config.json:
```
{
        "tleroot": "/tmp/tle/",
        "tlefile": "noaa.txt",
        "mapfile": "/usr/local/share/arftracksat/map.json",

        "tlesources": [
                "http://celestrak.com/NORAD/elements/weather.txt",
                "http://celestrak.com/NORAD/elements/noaa.txt"
        ],

        "updatePerdiod": 100,

        "station": {
                "name": "P-315",
                "lat": 41.8583,
                "lon": 2.2923,
                "hgt": 69.0
        },

        "show": [],

        "columns": [
                "name",
                "azel",
                "dis",
                "geo",
                "tab",
                "pos",
                "vel"
        ]
}
```

**Comment by Peiprjs** [comment]
```
    at /home/peipr/arftracksat/src/graphics.cpp:207
207	    auto selsat = shownSats[selsatidx];
(gdb) bt
#0  0x00005555555d66d2 in common2d ()
    at /home/peipr/arftracksat/src/graphics.cpp:207
#1  0x00005555555d9747 in render ()
    at /home/peipr/arftracksat/src/graphics.cpp:348
#2  0x00007ffff7db0d83 in ?? () from /lib/x86_64-linux-gnu/libglut.so.3
#3  0x00007ffff7db4619 in fgEnumWindows ()
   from /lib/x86_64-linux-gnu/libglut.so.3
#4  0x00007ffff7db12fb in glutMainLoopEvent ()
   from /lib/x86_64-linux-gnu/libglut.so.3
#5  0x00007ffff7db1bb5 in glutMainLoop ()
   from /lib/x86_64-linux-gnu/libglut.so.3
#6  0x00005555555d98d7 in startGraphics ()
    at /home/peipr/arftracksat/src/graphics.cpp:395
#7  0x0000555555606f0e in std::__invoke_impl<void, void (*)()> (
    __f=@0x55555567dd88: 0x5555555d97bf <startGraphics()>)
    at /usr/include/c++/10/bits/invoke.h:60
#8  0x0000555555606eb8 in std::__invoke<void (*)()> (
    __fn=@0x55555567dd88: 0x5555555d97bf <startGraphics()>)
    at /usr/include/c++/10/bits/invoke.h:95
#9  0x0000555555606ddc in std::thread::_Invoker<std::tuple<void (*)()> >::_M_invoke<0ul> (this=0x55555567dd88) at /usr/include/c++/10/thread:264
#10 0x0000555555606c4e in std::thread::_Invoker<std::tuple<void (*)()> >::operator() (this=0x55555567dd88) at /usr/include/c++/10/thread:271
--Type <RET> for more, q to quit, c to continue without paging--
#11 0x0000555555606766 in std::thread::_State_impl<std::thread::_Invoker<std::tuple<void (*)()> > >::_M_run (this=0x55555567dd80)
    at /usr/include/c++/10/thread:215
#12 0x00007ffff7c549c4 in ?? () from /lib/x86_64-linux-gnu/libstdc++.so.6
#13 0x00007ffff7f8cd80 in start_thread (arg=0x7ffff63a2640)
    at pthread_create.c:481
#14 0x00007ffff794476f in clone ()
    at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
(gdb) bt
#0  0x00005555555d66d2 in common2d ()
    at /home/peipr/arftracksat/src/graphics.cpp:207
#1  0x00005555555d9747 in render ()
    at /home/peipr/arftracksat/src/graphics.cpp:348
#2  0x00007ffff7db0d83 in ?? () from /lib/x86_64-linux-gnu/libglut.so.3
#3  0x00007ffff7db4619 in fgEnumWindows ()
   from /lib/x86_64-linux-gnu/libglut.so.3
#4  0x00007ffff7db12fb in glutMainLoopEvent ()
   from /lib/x86_64-linux-gnu/libglut.so.3
#5  0x00007ffff7db1bb5 in glutMainLoop ()
   from /lib/x86_64-linux-gnu/libglut.so.3
#6  0x00005555555d98d7 in startGraphics ()
    at /home/peipr/arftracksat/src/graphics.cpp:395
#7  0x0000555555606f0e in std::__invoke_impl<void, void (*)()> (
    __f=@0x55555567dd88: 0x5555555d97bf <startGraphics()>)
    at /usr/include/c++/10/bits/invoke.h:60
#8  0x0000555555606eb8 in std::__invoke<void (*)()> (
    __fn=@0x55555567dd88: 0x5555555d97bf <startGraphics()>)
    at /usr/include/c++/10/bits/invoke.h:95
#9  0x0000555555606ddc in std::thread::_Invoker<std::tuple<void (*)()> >::_M_invoke<0ul> (this=0x55555567dd88) at /usr/include/c++/10/thread:264
#10 0x0000555555606c4e in std::thread::_Invoker<std::tuple<void (*)()> >::operator() (this=0x55555567dd88) at /usr/include/c++/10/thread:271
--Type <RET> for more, q to quit, c to continue without paging--c
#11 0x0000555555606766 in std::thread::_State_impl<std::thread::_Invoker<std::tuple<void (*)()> > >::_M_run (this=0x55555567dd80) at /usr/include/c++/10/thread:215
#12 0x00007ffff7c549c4 in ?? () from /lib/x86_64-linux-gnu/libstdc++.so.6
#13 0x00007ffff7f8cd80 in start_thread (arg=0x7ffff63a2640) at pthread_create.c:481
#14 0x00007ffff794476f in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
```
Aquí tienes tu backtrace

**Comment by arf20** [comment]
On it, thanks for reporting. Easy fix.

**Comment by arf20** [comment]
It should be fixed now, 9d09bb2. Again thanks for reporting.
